shlr/gdb/src/gdbserver/core.c

```diff
@@ -160,7 +160,7 @@ static int _server_handle_vCont(libgdbr_t *g, int (*cmd_cb) (void*, const char*,
 	if (!(action = strtok (g->data, ";"))) {
 		return send_msg (g, "E01");
 	}
-	while (action = strtok (NULL, ";")) {
+	while ((action = strtok (NULL, ";")) != 0) {
```

Comment:
i dont think action is an int. so remove that != 0 because is unecessary and wrong